collect_app/src/main/AndroidManifest.xml

```diff
@@ -79,7 +79,8 @@ the specific language governing permissions and limitations under the License.
         android:largeHeap="true"
         android:supportsRtl="true"
         android:usesCleartextTraffic="true"
-        android:allowBackup="true">
+        android:allowBackup="true"
+        android:theme="@style/Theme.AppCompat.Light">
```

Comment:
Why did you have to make this change?

Reply:
Hi @seadowg! Whenever I run the tests for the newly created fragment, it gives me the error `java.lang.IllegalStateException: You need to use a Theme.AppCompat theme (or descendant) with this activity.`

Reply:
Ah right. Yes this would fix it but it would also change the default theme for the whole app which could cause us some problems. 

What's happening is that a component in the `DialogFragment` relies on the `AppCompat` theme so we can make sure that the Activity being used in tests has that theme. You can call `setTheme(R.style.Theme_AppCompat)` on the Activity or use our `RobolectricHelpers.createThemedActivity` helper.

Reply:
Yes @seadowg, I was also concerned for the default theme of the app. so I removed light, but using `RobolectricHelpers.createThemedActivity` helper would be a better option.